contracts/lz/interfaces/ILayerZeroEndpoint.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.0;
+
+import "./ILayerZeroUserApplicationConfig.sol";
+
+interface ILayerZeroEndpoint is ILayerZeroUserApplicationConfig {
```

Comment:
Please, instead of creating a new file, let's import `ILayerZeroEndpoint` adding LayerZero repo as dependency